pom.xml

```diff
@@ -17,26 +17,13 @@
         <snakeyaml.version>2.0</snakeyaml.version>
     </properties>
     <build>
-        <pluginManagement>
-            <plugins>
-                <plugin>
-                    <groupId>org.apache.maven.plugins</groupId>
-                    <artifactId>maven-compiler-plugin</artifactId>
-                    <version>${maven-compiler-plugin.version}</version>
-                    <configuration>
-                        <source>${maven.compiler.source}</source>
-                        <target>${maven.compiler.target}</target>
-                    </configuration>
-                </plugin>
-            </plugins>
-        </pluginManagement>
```

Comment:
I think this block is redundant, at least for now

Reply:
it's better to keep with plugin version

Reply:
why is it required?

Reply:
since plugin version allows to control version

Reply:
Probably there is some misunderstanding, I'm talking about `pluginManagement` block. As I understand it's good to have, when we have multiple POM files (with inheritance), right?

> pluginManagement: is an element that is seen along side plugins. Plugin Management contains plugin elements in much the same way, except that rather than configuring plugin information for this particular project build, it is intended to configure project builds that inherit from this one. However, this only configures plugins that are actually referenced within the plugins element in the children. The children have every right to override pluginManagement definitions.

Reply:
yes, I was talking not about plugin management, rather about version tag.
Once you remove pluginmanagement containing this tag you should put in plugins dep for compiler plugin

Reply:
Oh, got it, sorry my bad. Done.

Reply:
Also changed java version to 11